SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/StaticSubobjectsTest/StaticSubobjectsTest.cpp

```diff
@@ -174,7 +208,16 @@ void AStaticSubobjectsTest::PrepareTest()
 	CheckClientSeeIntProperty(2);
 
 	// Step 17
-	CheckClientNumberComponentsOnTestActorWithWait(InitialNumComponents - 1);
+	AddStep(TEXT("StaticSubobjectsTestClientSeeRightNumberComponentsWithoutWait3"), FWorkerDefinition::Client(1), nullptr,
+		[this]() {
+			AssertIsValid(TestActor->TestStaticComponent1, TEXT("TestStaticComponent1 should be valid."));
```

Comment:
requires instead of asserts maybe in the super-rare case

Reply:
done in latest commit